pom.xml

```diff
@@ -134,41 +159,20 @@
       </modules>
     </profile>
   </profiles>
-<!--
+
   <repositories>
     <repository>
-      <id>foconis-release</id>
```

Comment:
entweder braucht Bamboo einen PAT in seinen settings.xml oder ich muss diese auf Profil-Ebene hochziehen, damit er die repos aus https://mvnrepo.foconis.de runterladen kann.
Wir können diskutieren welche Möglichkeit ihr da möchtet

Reply:
wir benutzen default -Pfoconis --> bamboo braucht nichts